README.md

```diff
@@ -7,62 +8,78 @@
 Redis Operator creates/configures/manages redis-failovers atop Kubernetes.
 
 ## Requirements
+
 Redis Operator is meant to be run on Kubernetes 1.8+.
 All dependecies have been vendored, so there's no need to any additional download.
 
 ### Versions deployed
+
 The image versions deployed by the operator can be found on the [constants file](operator/redisfailover/service/constants.go) for the RedisFailover service.
 
 ## Images
+
 ### Redis Operator
+
 [![Redis Operator Image](https://quay.io/repository/spotahome/redis-operator/status "Redis Operator Image")](https://quay.io/repository/spotahome/redis-operator)
 
 ## Operator deployment on kubernetes
+
 In order to create Redis failovers inside a Kubernetes cluster, the operator has to be deployed. It can be done with [deployment](example/operator) or with the provided [Helm chart](charts/redisoperator).
 
 ### Using a Deployment
+
 To create the operator, you can directly create it with kubectl:
+
 ```
 kubectl create -f https://raw.githubusercontent.com/spotahome/redis-operator/master/example/operator/all-redis-operator-resources.yaml
 ```
+
 This will create a deployment named `redisoperator`.
 
 ### Using the Helm chart
+
 From the root folder of the project, execute the following:
+
 ```
 helm install --name redisfailover charts/redisoperator
 ```
 
 ## Usage
+
 Once the operator is deployed inside a Kubernetes cluster, a new API will be accesible, so you'll be able to create, update and delete redisfailovers.
 
 In order to deploy a new redis-failover a [specification](example/redisfailover/all-options.yaml) has to be created:
+
 ```
 kubectl create -f https://raw.githubusercontent.com/spotahome/redis-operator/master/example/redisfailover/all-options.yaml
```

Comment:
In this PR the `example/redisfailover/all-options.yaml` file got remove, so:

1. Which example are we going to use? `example/redisfailover/basic.yaml`?
2. this URL needs to be updated to the example to be used.